variables.tf

```diff
@@ -355,8 +361,20 @@ variable "ssm_tag_patch_group" {
   description = "SSM Patching group for instances. For more information: https://cloud.cms.gov/patching-prerequisites"
 }
 
-variable "ssm_tag_patch_window" {
+variable "ssm_DevTestImpl_patch_window" {
   type        = string
   default     = "ITOPS-Wave1-Non-Mktplc-DevTestImpl-MW"
   description = "SSM Patching window for instances. For more information: https://cloud.cms.gov/patching-prerequisites"
 }
+
+variable "ssm_prod_patch_window" {
+  type        = string
+  default     = "ITOPS-Wave1-Non-Mktplc-Prod-MW"
+  description = "SSM Patching window for instances. For more information: https://cloud.cms.gov/patching-prerequisites"
+}
```

Comment:
Update the desc to differentiate between prod/nonprod

Reply:
Would it makes sense to hardcode these and just let the `override` override it?  I could go either way.